backend/app/Services/Grpc/GrpcGetter.php

```diff
@@ -1,23 +1,27 @@
 <?php
+
 declare(strict_types=1);
 
 namespace App\Services\Grpc;
-use GrpcClient\Nlp\GenerateAllRequest;
+
 use Grpc\ChannelCredentials;
+use GrpcClient\Nlp\GenerateAllRequest;
 use GrpcClient\Nlp\NlpManagerClient;
 
 class GrpcGetter
 {
-    public function getGrpcRequest(int $userId):bool
+    public function getGrpcRequest(int $userId): bool
     {
-        $client = new NlpManagerClient(config('grpc.server_url'),
-        [
-            'credentials' => ChannelCredentials::createInsecure(),
-        ]
-    );
+        $client = new NlpManagerClient(
+            config('grpc.server_url'),
```

Comment:
**[phpstan]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Parameter #1 $hostname of class GrpcClient\Nlp\NlpManagerClient constructor expects string, mixed given.